.github/workflows/backend-build-test.yml

```diff
@@ -42,7 +42,7 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # 4.1.7
```

Comment:
hi recommends using the major version eg `@v2 @v3`, not the commit hash version

Reply:
got it